release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -60,6 +60,71 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-6-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-6-live-env-with-coreos-installer"]
+==== {op-system} PXE and ISO now live environment
+
+The PXE media and ISO available for {op-system} are now a fully live
+environment. Unlike the previous dedicated PXE media and ISO used for
+installation, the {product-title} live environment can be configured with
+Ignition and contains all the same packages as the main {op-system}, such as
+podman. This allows arbitrary scripting of pre/post installation workflows. For
+example, you could run `coreos-installer` and then make an HTTP request to
+signal success to a provisioning server. PXE boots use the normal
+`ignition.config.url`. The ISO can be configured with Ignition by using the
+following command:
+
+[source,terminal]
+----
+$ coreos-installer iso ignition embed
+----
+
+[id="ocp-4-6-coreos-installer-rewritten"]
+==== `coreos-installer` has been rewritten
+
+The `coreos-installer` has been rewritten to support more features, some of
+which include:
+
+* Modifying the kernel arguments of the installed system.
+* Fetching Ignition configs.
```

Comment:
s/configs/configuration files

Reply:
"Ignition config" is the common term; I think "Ignition configuration file" will be more confusing here.

Reply:
@ahardin-rh our [glossary of terms](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/term_glossary.adoc#ignition-config) and the majority of our docs refer to it as "Ignition config." Do you agree with keeping it as-is?

Reply:
If it's more common, that's fine by me!